mm2src/coins/utxo/utxo_common.rs

```diff
@@ -2087,18 +2167,44 @@ where
     }
 }
 
-pub fn check_all_utxo_inputs_signed_by_pub(
+pub fn check_all_utxo_inputs_signed_by_pub<T: UtxoCommonOps>(
```

Comment:
I think it's good to add a comment why this fn is needed (and that it expects uncompressed pubkeys)

Reply:
[7b24505](https://github.com/KomodoPlatform/komodo-defi-framework/pull/2408/commits/7b245057241d2faedb1de279d62112898fa81f85)

Reply:
Thanks great!
(btw iirc there was a discussion that checking that at least one input is signed with the pk would be sufficient for this)